R/functions_utility_interaction.R

```diff
@@ -0,0 +1,912 @@
+####################### DATA PROCESSING ########################################
```

Comment:
I wonder whether this script name etc could get confused with "interaction" effects? I recognise that it might be difficult to turn the ship now though, @marion-hoffman, but maybe even just renaming the script functions_group_interaction.R might help?